lib/engine/game/g_18_uruguay/step/destination_bonus.rb

```diff
@@ -12,24 +12,27 @@ def description
           end
 
           def log_skip(entity)
-            return if entity.minor?
+            return unless entity.corporation?
             return if entity == @game.rptla
+            return if @game.abilities(entity, :destination_bonus).nil?
+            return if destination_node_check?(entity).nil?
```

Comment:
```suggestion
            return unless destination_node_check?(entity)
```

1. You're checking for a destination bonus ability in `destination_node_check?` so you don't need to repeat that test here.
2. A method that has a question mark in its name should only return `true` or `false`, so testing `nil?` looks wrong.
3. It would be better to change the guard clauses in `destination_node_check?` from `return if …` to `return false if …`.